arch/x86/core/Kconfig.intel64

```diff
@@ -0,0 +1,42 @@
+# Kconfig.intel64 - Intel64-specific X86 subarchitecture options
+#
+# Copyright (c) 2019 Intel Corp.
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Know you're just copying the format in other files, but it feels like this extra `#` started out as someone's editor trying to be helpful by inserting an extra `#` when they pressed enter, until they pressed enter again. Maybe it could be removed...

Reply:
Removed :)